src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -861,19 +873,13 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('failover_policy', help="The failover policy of the Failover Group",
                    arg_type=get_enum_type(FailoverPolicyType))
         c.argument('grace_period',
-                   help='Interval in hours before automatic failover is initiated '
-                        'if an outage occurs on the primary server. '
-                        'This indicates that Azure SQL Database will not initiate '
-                        'automatic failover before the grace period expires. '
-                        'Please note that failover operation with AllowDataLoss option '
-                        'might cause data loss due to the nature of asynchronous synchronization.')
+                   arg_type=grace_period_param_type)
         c.argument('add_db', nargs='+',
                    help='List of databases to add to Failover Group')
         c.argument('remove_db', nargs='+',
                    help='List of databases to remove from Failover Group')
         c.argument('allow-data-loss',
-                   help='Complete the failover even if doing so may result in data loss. '
-                        'This will allow the failover to proceed even if a primary database is unavailable.')
+                   arg_type=allow_data_loss_param_type)
```

Comment:
yes it is a bool.I can update this but I was keeping it consistent with the sql az failover-group cmd, which also has allow_data_loss. these two are sibling cmds and I am not sure what the best approach is here. Can't update the existing cmd to match allow_data_loss either as it will break existing customers

Reply:
The current command looks fine, so I think we are probably good.

```
(env) D:\git\azure-cli [dev ≡]> az sql failover-group set-primary -h

Command
    az sql failover-group set-primary : Set the primary of the failover group by failing over all
    databases from the current primary server.

Arguments
    --name -n [Required] : The name of the Failover Group.
    --allow-data-loss    : If specified, the failover operation will allow data loss.
```

Can you post similar help output for the new command to confirm that it's consistent?